components/VerifyPresentationResult.yml

```diff
@@ -19,72 +19,303 @@ components:
+                      wasChecked:
+                        type: boolean
+                        description: Whether this VerifiableCredential was actually verified. If false, detailed verification properties will not be present.
+                      verified:
+                        type: boolean
+                        description: Whether this specific VerifiableCredential pass verification. Only meaningful if wasChecked is true.
+                      skipReason:
+                        type: boolean
+                        description: Explanation of why verification was skipped for this VerifiableCredential. Only present if wasChecked is false.
+                      validFrom:
+                        type: object
+                        description: Results of validating the validFrom property, if present in the verifiableCredential. Only present if wasChecked is true.
+                        properties:
+                          valid:
+                            type: boolean
+                            description: Validation result.
+                          input:
+                            type: string
+                            description: The validFrom value.
+                      validUntil:
+                        type: object
+                        description: Results of validating the validUntil property, if present in the verifiableCredential. Only present if wasChecked is true.
+                        properties:
+                          valid:
+                            type: boolean
+                            description: Validation result.
+                          input:
+                            type: string
+                            description: The validUntil value.
+                      credentialSchema:
+                        type: array
+                        description: Results from validating the credentialSchema objects if any. Only present if wasChecked is true.
+                        items:
+                          type: object
+                          description: Results from validating a credentialSchema object.
+                          properties:
+                            valid:
+                              type: boolean
+                              description: Results of validating the credentialSchema object.
+                            input:
+                              type: string
+                              description: The credentialSchema object.
+                      credentialStatus:
+                        type: array
+                        description: Results from validating the credentialStatus objects if any. Only present if wasChecked is true.
+                        items:
+                          type: object
+                          description: Results of validating a credentialStatus object.
+                          properties:
+                            value:
```

Comment:
Hmm, this seems like you would need to pass in the expected status in the check... I don't think we can report on 'valid' or not without that. We might need something like: `expectedValue`.